pkg/interface/src/views/apps/links/components/LinkDetail.tsx

```diff
@@ -27,12 +27,13 @@ export function LinkDetail(props: LinkDetailProps) {
   return (
     /*  @ts-ignore indio props?? */
     <Row height="100%" width="100%" flexDirection={['column', 'column', 'row']} {...rest}>
-      <LinkBlockItem minWidth="0" minHeight="0" height={['50%', '50%', '100%']} width={['100%', '100%', 'calc(100% - 350px)']} flexGrow={0} border={0} node={node} />
+      <Center flexBasis={['100%', '100%', '75%']} flexShrink={1} overflowY="scroll" >
```

Comment:
You added a `Center` here, but there's another place where `LinkBlockItem` is used that needs it as well.

Reply:
The summary view is an interesting case; I'm not immediately sure what the best approach is. For now, I was able to address it by adding a flex container inside the box: see 72efd56b0b and 2f08d07527. 

![image](https://user-images.githubusercontent.com/57736583/163860143-4d7ad985-d0c0-42ec-a021-807dcd743616.png)